ndnfs/driver/driver.go

```diff
@@ -0,0 +1,367 @@
+package driver
```

Comment:
Since it looks like a copy of 'ndnfs/daemon/driver.go' the same remarks are applied to this file as I made for the 'ndnfs/daemon/driver.go'